tomopy_cli/file_io.py

```diff
@@ -63,6 +57,22 @@ def read_tomo(sino, params):
 
     return proj, flat, dark, theta, rotation_axis
 
+
+def blocked_view(proj, theta, params):
+    log.info("  *** correcting for blocked view data collection")
+    if params.blocked_views:
+        log.warning('  *** *** ON')
```

Comment:
For global (and local) logging: https://stackoverflow.com/questions/7621897/python-logging-module-globally

Reply:
I think I am already using a  global logger. This is defined in [log.py](https://github.com/tomography/tomopy-cli/blob/master/tomopy_cli/log.py).

Reply:
ah you mean having tomopy and tomopy-cli using the same logging?